geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/JdbcDUnitTest.java

```diff
@@ -235,9 +235,28 @@ public void getReadsFromDB() throws Exception {
       region.invalidate(key);
 
       PdxInstance result = (PdxInstance) region.get(key);
-      assertThat(result.getField("id")).isEqualTo(pdxEmployee1.getField("id"));
-      assertThat(result.getField("name")).isEqualTo(pdxEmployee1.getField("name"));
-      assertThat(result.getField("age")).isEqualTo(pdxEmployee1.getField("age"));
+      assertThat(result.getFieldNames().size()).isEqualTo(3);
```

Comment:
Better syntax (with better AssertionError message) is:
```java
assertThat(result.getFieldNames()).hasSize(3);
```
If the size doesn't match the above will list everything in the collection or array.